stable/kubernetes-dashboard/values.yaml

```diff
@@ -0,0 +1,22 @@
+# Default values for kubernetes-dashboard
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+image: gcr.io/google_containers/kubernetes-dashboard-amd64
+imageTag: "v1.6.0"
+imagePullPolicy: "IfNotPresent"
+
+namespace: kube-system
+
```

Comment:
AFAIK the dashboard is usually deployed in `kube-system` by convention, though there's no technical requirement that it has to be there. Cluster operators can and do install cluster-service charts in any namespace. The [core-dns chart](https://github.com/kubernetes/charts/blob/5348297977b97b6c7599cde465f61e499c7ef5e5/stable/coredns/README.md) install example specifies the namespace from the command line, as opposed to it being enforced by the chart itself.